modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_single_card.yml

```diff
@@ -0,0 +1,60 @@
+id: az_paragraph_single_card
+label: AZ Paragraph Single Card
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_single_card
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_cards
+
+process:
+
+  field_az_cards/title: 
+    plugin: extract
+    source: field_uaqs_short_title
+    index:
+      - 0
+      - value
+
+  field_az_cards/body:
+    plugin: extract
+    source: field_uaqs_summary
+    index:
+      - 0
+      - value
+
+  field_az_cards/link_title:
+    plugin: extract
+    source: field_uaqs_links
+    index:
+      - 0
+      - title
+
+  field_az_cards/link_uri:
```

Comment:
Does it matter if line 39 says "uri" and line 44 says "url"?

Reply:
Cards field is `link_uri` only.